src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -1,23 +1,602 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-    <PropertyGroup>
-      <TargetFrameworks>$(NetMinimum);$(NetMinimum)-unix;$(NetMinimum)-windows;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
-      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-      <StrongNameKeyId>Microsoft</StrongNameKeyId>
-      <IsPackable>false</IsPackable>
-      <VersionPrefix>4.8.6</VersionPrefix>
-      <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.9.0</VersionPrefix>
-      <AssemblyVersion>4.6.1.6</AssemblyVersion> <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
-      <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.6.1.6</AssemblyVersion>
-      <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.6.2.0</AssemblyVersion>
-      <PackageValidationBaselineVersion>4.8.6</PackageValidationBaselineVersion>
-      <UseTargetFrameworkPackage>true</UseTargetFrameworkPackage>
-    </PropertyGroup>
-
-    <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
-    <PropertyGroup>
-      <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
-      <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' != 'windows'">SR.PlatformNotSupported_System_Data_SqlClient</GeneratePlatformNotSupportedAssemblyMessage>
-    </PropertyGroup>
-
-  </Project>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetMinimum);$(NetMinimum)-unix;$(NetMinimum)-windows;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.8.6</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.9.0</VersionPrefix>
+    <AssemblyVersion>4.6.1.6</AssemblyVersion>
+    <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
```

Comment:
Do we actually need to freeze this one?  I think it's officially OOB everywhere and the package should win in all cases.  If that's the case then we should remove the special cases here and make it look more like the `Microsoft.BCL` project.